### PR TITLE
Fix linting warnings

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-message bundle size difference from 4.1.3 to 4143b4faaed6e37ff3df5dc89ef2f4fbf3ccfecd
No significant bundle size differences found.

File: src/js/message.js [hunk 1]
```diff
@@ -1,12 +1,35 @@
 import construct from './construct-element';
 
+/**
+ * A number, or a string containing a number.
+ * @typedef {Object} MessageOptions
```
Comment (JakeChampion) [inline src/js/message.js:5]: 🙌

File: src/js/message.js [hunk 2]
```diff
@@ -95,6 +119,7 @@ class Message {
 
 	/**
 	 * Open the message.
+	 * @returns {undefined}
```
Comment (JakeChampion) [inline src/js/message.js:122]: This function does not return an explicit undefined, which is what this return documentation is stating.

To document an implicit undefined you need to use `void`, according to what my editor is telling me anyway

```suggestion
	 * @returns {void}
```

Comment (notlee) [reply src/js/message.js]: cool thanks, done :) I had a quick look at the `@returns` docs and couldn't see anything for implicit undefined returns

**Comment by origamiserviceuser** [comment]
o-message bundle size difference from 4.1.3 to 6a09c15f352635111ca4d726f32ac9dabee4e41d
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v4.1.4 :tada:

**Pull request merged**